### ISSUE TITLE
Petit soucis | Usebug

### ISSUE THREAD

**Issue by Ryload** [indilo53/fivem-esx_weedjob#6]
Salut Indilo ! Je viens te signaler un bug plutôt gênant ! En appuyant sur entrée pour récolter, ça lance le timer et boom au bout de x secondes, tu as 1 de weed. Et bien en appuyant sur entrée, en sortant de la zone, rappuyer sur entrer, et du coup tu récoltes énormement d'un coup ( un joueur à fait plus de 10.000 de weed très rapidement en une soirée sur mon serveur pour te dire ^^ )

Saurais-tu check ça ?

**Comment by ig0ne** [comment]
même soucis peut être jouer sur le SetTimeout en le plaçant de façon à ce que quand le joueur sorte de la zone il ne puisse pas récolter à nouveau avant un certain temps:

**Comment by Ryload** [comment]
Yep, aucune idée. De même pour le nouveau systeme de cops requis. 

Un "object is nil" ressort ligne 29 : 	if CopsConnected < Config.RequiredCops

Hmmm

**Comment by indilo53** [comment]
As tu mis à jour ton fichier config.lua ?

_Sent from my Samsung SM-G920F using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Ryload** [comment]
Yep sir

**Comment by indilo53** [comment]
Je viens de faire une maj pour le souci de CopsConnected
